docs/source/ecf.rst

```diff
@@ -162,6 +169,14 @@ As we want to do our own spectral extraction, we set this variable to ``calints`
 	Note that other Instruments might used different suffixes!
 
 
+horizonsfile
+''''''''''''
+Only used for HST analyses. This should be set to the fully qualified path to the horizons file you've downloaded from https://ssd.jpl.nasa.gov/horizons/app.html#/. On that website, 1. Select "Vector Table", 2. Select "HST", 3. Select "@ssb" (Solar System Barycenter), 4. Select a date range that spans the days relevant to your observations. Then click Generate Ephemeris and click Download Results.
```

Comment:
Since downloading the files isn't too complicated, I'm personally in favour of leaving it to others to download as it'll be one less thing that we'll have to regularly maintain (especially if someone's working on brand new observations).